VotingApplication/VotingApplication.Web/Scripts/Directives/VotingStrategy.js

```diff
@@ -1,9 +1,10 @@
 ﻿(function () {
-    angular.module('GVA.Voting').directive('votingStrategy', ['PollService', function (PollService) {
+    angular.module('GVA.Voting').directive('votingStrategy', ['PollService', '$routeParams', function (PollService, $routeParams) {
```

Comment:
This is rather a long line. Would be a bit easier on the eyes if .module and .controller were each on their own line.